src/shared-components/DataGridWrapper/DataGridWrapper.tsx

```
@@ -0,0 +1,122 @@
+        pageSizeOptions={paginationConfig?.pageSizeOptions ?? [10, 25, 50, 100]}
+        onRowClick={onRowClick ? (p) => onRowClick(p.row as T) : undefined}
+        autoHeight
+        disableRowSelectionOnClick
+      />
+      {error && (
+        <div role="alert" className={styles.errorMessage}>
+          {error}
+        </div>
+      )}
+      {!loading && !error && filtered.length === 0 && (
+        <div role="status">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)